apifest-oauth20/src/test/java/com/apifest/oauth20/AuthorizationServerTest.java

```diff
@@ -1097,15 +1100,16 @@ public void when_issue_auth_code_check_scope_valid() throws Exception {
 
         given(req.getUri()).willReturn(
                 "http://example.com/oauth20/authorize?redirect_uri=http%3A%2F%2Fexample.com&response_type=code&client_id="
-                        + clientId);
+                        + clientId + "?state=xyz");
         willReturn("basic").given(authServer.scopeService).getValidScope(null, clientId);
 
         // WHEN
-        authServer.issueAuthorizationCode(req);
+        String response = authServer.issueAuthorizationCode(req);
 
         // THEN
         verify(authServer).generateCode();
         verify(authServer.scopeService).getValidScope(null, clientId);
+
```

Comment:
Please, add a check that verifies the state is returned in the response. Also, it is better to have a separate test that checks exactly that.